widget_driver/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 0.2.0
```

Comment:
As this is a breaking API Change, we should bump the major version (meaning also to jump to 1.0.0 wohooo!!) What do you think about this?